main.nf

```diff
@@ -6,162 +6,195 @@
  nf-core/epitopeprediction Analysis Pipeline.
  #### Homepage / Documentation
  https://github.com/nf-core/epitopeprediction
+ #### Authors
+ Christopher Mohr christopher-mohr <christopher.mohr@qbic.uni-tuebingen.de> - https://github.com/christopher-mohr>
+ Alexander Peltzer apeltzer <alexander.peltzer@qbic.uni-tuebingen.de> - https://github.com/apeltzer
 ----------------------------------------------------------------------------------------
 */
 
-
 def helpMessage() {
+    log.info nfcoreHeader()
     log.info"""
-    =========================================
-     nf-core/epitopeprediction v${workflow.manifest.version}
-    =========================================
+
     Usage:
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/epitopeprediction --reads '*_R{1,2}.fastq.gz' -profile standard,docker
+    nextflow run nf-core/epitopeprediction --somatic_mutations '*.vcf.gz' --alleles '*.alleles' -profile standard,docker
```

Comment:
```suggestion
    nextflow run nf-core/epitopeprediction --somatic_mutations "*.vcf.gz" --alleles "*.alleles" -profile docker
```

Reply:
Can you be more consistent with the example/default command line to launch the pipeline?